setup-wizard.yml

```diff
@@ -1,64 +1,5 @@
 version: "2"
 fields:
-  - id: config_mode
-    title: Execution Client Config Mode
-    description: >-
-      Advanced and Development Use Only: Please only enable this option if you are confident you know what you are doing, if you have any doubts, please use the basic setting.
-      
-      Set this to **"advanced"** to pass in an arbitrary Execution Client Engine-API or leave on **basic** to choose one below.
-    target:
-      type: environment
-      name: CONFIG_MODE
-      service: beacon-chain
-    enum:
-      - "basic"
-      - "advanced"
-  - id: HTTP_ENGINE
-    required: true
-    target:
-      type: environment
-      name: HTTP_ENGINE
-      service: beacon-chain
-    title: Execution Client Endpoint
-    description: >-
-      The Consensus Clients (Lighthouse, Lodestar, Nimbus, Prysm, and Teku) require access to an Execution Client to receive the execution payload by using the Engine API of the Execution Client.   
-
-      This endpoint MUST be provided for an Execution Client that is run locally.   
-
-      IT IS NO LONGER POSSIBLE TO USE INFURA OR ANY OTHER SERVICE PROVIDER AS YOUR EXECUTION CLIENT.
-
-      - Geth [Install link](http://my.dappnode/#/installer/geth.dnp.dappnode.eth)
-
-      - Nethermind [Install link](http://my.dappnode/#/installer/nethermind.public.dappnode.eth)
-
-      - Erigon [Install link](http://my.dappnode/#/installer/erigon.dnp.dappnode.eth)
-
-
-      Consensus Clients require access to an Execution Client in order to operate properly.
-      Be sure to only select an Execution Client below that you have already installed.
-    enum:
-      - "http://geth.dappnode:8551"
-      - "http://nethermind.public.dappnode:8551"
-      - "http://erigon.dappnode:8551"
-    if: { "config_mode": { "enum": ["basic"] } }    
-  - id: HTTP_ENGINE
```

Comment:
I guess this was just outdated, all other packages get this from global env

Reply:
yes it was outdated from pre-global ENVs its how alll CLs worked for a short time before stakersUI was implemented on mainnet, but work was stopped midway, since at the time lodestar still didnt support the Web3Signer, and Lion told us to hold off and focus on other projects until lodestar was ready